virtualizarr/tests/__init__.py

```diff
@@ -39,6 +39,7 @@ def _importorskip(
 has_imagecodecs, requires_imagecodecs = _importorskip("imagecodecs")
 has_hdf5plugin, requires_hdf5plugin = _importorskip("hdf5plugin")
 has_zarr_python, requires_zarr_python = _importorskip("zarr")
+has_obstore, requires_obstore = _importorskip("obstore")
```

Comment:
To support #353 and #473 `obstore` would need to become a required dependency

Reply:
I think I'd rather it become required at that point rather than in this PR, but it's not a strong preference.